src/Adapter/Assets/AssetUrlGeneratorTrait.php

```diff
@@ -40,8 +40,11 @@ protected function getUriFromPath($fullPath)
 
     protected function getPathFromUri($fullUri)
     {
-        if ('' !== ($trimmedUri = rtrim($this->configuration->get('__PS_BASE_URI__'), '/'))) {
-            return $this->configuration->get('_PS_ROOT_DIR_').preg_replace('/\\'.$trimmedUri.'/', '', $fullUri, 1);
+        $trimmedUri = rtrim($this->configuration->get('__PS_BASE_URI__'), '/');
+
+        if ('' !== $trimmedUri) {
+            $uri = str_replace('/', '\\/', $trimmedUri);
+            return $this->configuration->get('_PS_ROOT_DIR_').preg_replace('/'.$uri.'/', '', $fullUri, 1);
```

Comment:
Good catch, I succeeded to reproduce the issue and your change fixes it.

However, I would like to avoid a `str_replace()` AND a `preg_replace()` if a lighter solution is possible for obvious performance reasons.
In order to not have a missing slash crashing the shop, we just have to change the delimiter of the regex (= its first and last characters).
We could chose `. By reusing the previous version we would get:
```php
if ('' !== ($trimmedUri = rtrim($this->configuration->get('__PS_BASE_URI__'), '/'))) {
    return $this->configuration->get('_PS_ROOT_DIR_').preg_replace('`\\'.$trimmedUri.'`', '', $fullUri, 1);
}
```